.circleci/config.yml

```diff
@@ -56,10 +88,14 @@ workflows:
           filters:
             tags:
               only: /.*/
+      - smoke_test:
+          requires:
+            - test
       - publish_github:
           context: Honeycomb Secrets for Public Repos
           requires:
             - test
```

Comment:
we can skip requiring test, since it's a linear pipeline test --> smoke --> publish
```suggestion
```

Reply:
Oh good point, removed 👍